Add new benchmark: Basque bench

zxcvuser opened the pull request:
BasqueBench is a benchmark for tasks in Basque that cover several evaluation areas. The datasets consist of professional translations of relevant English datasets and newly created datasets in Basque. The README.md contains detailed information on all the tasks included in the benchmark.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=2153) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=2153) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/EleutherAI/lm-evaluation-harness?pullRequest=2153) it.</sub>

baberabb commented:
Thanks very much for this PR. Just some small issues I identified and if you could also run 
```bash
pre-commit run --all-files
```
to fix the linting issues

haileyschoelkopf commented on lm_eval/tasks/basque_bench/flores_eu/create-yamls_flores_eu.py:
```diff
@@ -0,0 +1,115 @@
+"""
+Script to generate task YAMLs for the FLORES-200 dataset.
+Based on `tasks/translation/utils.py`.
```
Could we remove the hyphen in these yaml-generation scripts? **Hyphens are not allowed in python filenames.**

zxcvuser commented:
These are the changes done:
- Added the benchmark info in lm_eval/tasks/README.md
- Replaced "-" by "_" in the create_files script in flores_eu and added weight_by_size: false
- Run linters
- Remove grouping in mgsm and copa tasks (they were pointing to pre-existing benchmarks)
With these, it should all be fine now. Thank you!
